report/1.Introduction/1.IntroductionCtrl.tex

```diff
@@ -8,51 +8,53 @@
 \chapter{Introduction}
 
 Tracking airborne objects can be quite difficult.
-This difficulty has inspired different sports and hobbies, including skeet shooting.
-Hitting a target requires the tracking of it, it's velocity and direction, as well as the calculation or prediction of its future trajectory.
-The trajectory can be impacted by a number of different features, including gravity, wind, initial force, any artificial acceleration.
-Furthermore, if the object is intelligent in anyway, the path might change procedurally.
-
+This difficulty has inspired different sports and hobbies such as skeet shooting.
+Hitting a target requires tracking of it, its velocity and direction, as well as the calculation or prediction of its future trajectory.
+The trajectory can be impacted by a number of different factors, including gravity, wind, initial force, or any artificial acceleration.
+Furthermore, if the object is intelligent in any way, the path might change procedurally.
+%TODO procedurally is a bad word for this, isn't it?
```

Comment:
%TODO procedurally is a bad word for this, isn't it?

Reply:
accepted